bears/r/FormatRBear.py

```diff
@@ -68,20 +68,20 @@ def create_arguments(filename, file, config_file,
         rcode = ('library(formatR);'
                  'formatR::tidy_source('
                  'source="' + escape(filename, '"\\') + '",'
-                 'comment={r_keep_comments},'
                  'blank={r_keep_blank_lines},'
-                 'arrow={r_use_arrows},'
                  'brace.newline={r_braces_on_next_line},'
                  'indent='
                  '{tab_width}'.format(tab_width=tab_width,
-                                      r_keep_comments=_map_to_r_bool(
-                                          r_keep_comments),
                                       r_keep_blank_lines=_map_to_r_bool(
                                           r_keep_blank_lines),
-                                      r_use_arrows=_map_to_r_bool(
-                                          r_use_arrows),
                                       r_braces_on_next_line=_map_to_r_bool(
                                           r_braces_on_next_line)))
+        # Disable r_keep_comments if None
+        rcode += ('' if r_keep_comments == None
+                  else ',comment=' + _map_to_r_bool(r_keep_comments))
+        # Disable r_use_arrows if None
+        rcode += ('' if r_use_arrows == None
+                  else ',arrow=' + _map_to_r_bool(r_use_arrows))
         # Disable r_max_expression_length if it is equal to 0
         rcode += ('' if r_max_expression_length == 0
                   else ',width.cutoff=' + str(r_max_expression_length)) + ')'
```

Comment:
I kept the `None` because, IMO there is no need passing an option to tidy_source unless it is different from the default.
If you agree with this (i will do a simple modification on what i just pushed)